daisy_integration_tests/create_random_data_image.subwf.json

```diff
@@ -0,0 +1,105 @@
+{
+  "Name": "create-random-data-image",
+  "Vars": {
+    "size": {
+      "Required": true
+    },
+    "export_instance_disk_image": {
+      "Value": "projects/compute-image-tools/global/images/family/debian-9-worker",
+      "Description": "image to use for the worker instance"
+    },
+    "export_network": {
+      "Value": "global/networks/default",
+      "Description": "Network to use for the worker instance"
+    },
+    "export_subnet": {
+      "Value": "",
+      "Description": "SubNetwork to use for the worker instance"
+    }
+  },
+  "Sources": {
+    "create_random_data_disk.sh": "./create_random_data_disk.sh"
+  },
+  "Steps": {
+    "setup-disks": {
+      "CreateDisks": [
+        {
+          "Name": "disk-${NAME}",
+          "SourceImage": "${export_instance_disk_image}",
+          "Type": "pd-ssd"
+        },
+        {
+          "Name": "disk-data-${NAME}-${ID}",
+	  "SizeGb": "${size}",
+          "Type": "pd-ssd",
+	  "ExactName": true,
+	  "NoCleanup": true
+        }
+      ]
+    },
+    "create-random-data-disk": {
+      "CreateInstances": [
+        {
+          "Name": "inst-${NAME}",
+          "Disks": [{"Source": "disk-${NAME}"},{"Source": "disk-data-${NAME}-${ID}"}],
+          "MachineType": "n1-highcpu-4",
+          "Metadata": {
+            "block-project-ssh-keys": "true",
+	    "disk-size": "${size}",
+	    "disk-data-name": "disk-data-${NAME}-${ID}",
+	    "image-destination": "image-export-given-size-${size}gb"
+          },
+          "networkInterfaces": [
+            {
+              "network": "${export_network}",
+              "subnetwork": "${export_subnet}"
+            }
+          ],
+          "Scopes": ["https://www.googleapis.com/auth/devstorage.full_control", "https://www.googleapis.com/auth/compute"],
```

Comment:
For this simple script you should need no special scopes, its just filling up a disk then shuting down